src/db/tasks/create.ts

```diff
@@ -0,0 +1,3 @@
+import runOnConnection, { AvailableOperations } from "./abstract/RunOnClient";
+
+runOnConnection(AvailableOperations.CREATE);
```

Comment:
Just reading this function call does not make it clear to me exactly _what_ we are creating...

```suggestion
runOnConnection(AvailableOperations.CREATE_DB);
```